### DISCUSSION TITLE
WG - Open Source in Business Collaborative Project

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#481] [Community Proposals]
>[!NOTE]
> The TODO Steering Committee approved the creation of a working folder and working group under TODO to initiate async collaboration to the open source in business whitepaper/guide
>  Working repository here: https://github.com/todogroup/ospology/tree/main/whitepapers/business-value

During the [Touchpoint call on Thursday, May 16th](https://github.com/todogroup/ospology/discussions/478), participants presented the idea of building a collaborative project focused on open source in business. Below are the details of this proposal, including the motivations, potential assets the final output could cover, and open questions/discussions related to the workflow, the process to follow, and potential partnerships with enterprise leaders outside the open-source community.

### Background
- “Principles for Industrial Open Source” from Carl-Eric Mols (https://drive.google.com/file/d/19yjU5w9L0DAS_9Si-AmZmZYiEv3hdqiO/view )
- Presentation of Magnus Feuer at FOSSNorth (slide 8-10): https://foss-north.se/2024/slides/slides_mfeuer.pdf “How do we set the stage for FOSS projects driven by the community?”
- Engineering driven vs. Business driven
   - Engineering driven : well covered – BUT potentially rather considered as cost driver as it is also potentially re-active /PUSH
Business driven: ?
   - Mozilla archetypes: A Practical Framework For Open Source Strategic Success: https://opensource.com/open-organization/18/9/mozilla-open-archetypes 
   - Heather Meeker: From Project to Profit https://heathermeeker.com/from-project-to-profit/ 
              - Playbook: https://github.com/heathermeeker/COSSPlaybook/blob/main/Playbook.MD 
- Need to have a business perspective to address the business level in the organizations (e.g. the budget owners)

### Potential assets to be provided as final outcome
- Collection of success stories
- Business blueprints / case studies where an Open Source strategy helps to build ecosystems around a product/service and thus supports the business strategy enrich a product/service
- Enabling /Training/ Awareness
   - Enable business managers to create a business model that leverages Open Source as “strategic tool”
   - show ROI potential for invests in Open Source projects
   - explain the necessity of long-term investments (3-5 years) to build a community
   - Enable business managers to judge when Open Source as a “strategic tool” can help to support the business strategy AND when Open Source is not the right strategic tool 
- Side-by-side comparison: what is the proprietary alternative? (E.g. concerning “testing”/Quality Assurance / Strength and weaknesses on each side to provide base for decision)
- How to position the topic OpenSource in companies that are no “Software Natives” but rather Hardware/Thing companies/organizations
   - Current thinking: Feature, feature, feature – at the end the project is late nevertheless
   - Upfront investment instead in Open Source
- Ecosystem with dependencies on suppliers (vendor lock-in)?
- Relation Open Source to business models?
    - No use of Open Source without permission – thinking?
    - Missing link
    - Need material to explain and convince the management level (Why should I pay for Open Source – it is free)
    
By having better awareness on business side, Open Source initiatives as business enabler will potentially be requested as pro-active / PULL, where applicable.

- Number of Software Development?
   - Maintenance costs for applications, trainings etc. (Total cost of Ownership?) are typically not budgeted but only the development
          - And thus comparison with Open Source will be hard
    - Technical debt sucking up the resources needed for new features
    - The “hidden” costs for commercial software is also often not transparent
Models for calculating Software Costs?

### Logistics / Process:
Where to place this? OSPO book vs New material (guide / whitepaper)

- Comment: given the fact that the core scope is business value, we could integrate the material created into the OSPO book later
- Starting with a 1-page pitch deck as a first task to complete
- Comment: ​​Is there business oriented people / body we could partner with to co-publish with so we get past the "OSPO people will always say open source is the answer" problem? (Example: https://osb-alliance.de/)
    - Maybe a palace to include people outside open source that endorses the content created (CEO’s , CTO’s, etc)

## What's next?

If you are interested in participating, please: 
- React to this comment to show your interest using 👍
- Share your ideas and feedback in this thread
- Join the next Touchpoint call (either the thursday one or Friday one), scheduled on:

| Thursday (Community and General Members) | Friday (General Members) |
| --- | --- |
| June 27 12:00 – 13:00 UTC |June 14 15:30 – 16:30 UTC |

During the next Touchpoint call, the participants will review the comments shared during these weeks, and the final proposal will be sent to the Steering Committee to sing-off the scope, workflow and process to adopt

* _Reminder: You don't need to be a member to contribute to TODO resources or this new proposal. Everyone carries the water and helps out!_

**Comment by victorjunlu** [comment]
This is a great initiative.  As Leslie suggested in the meeting, partnering with other professional groups could bring visibility to TODO group and OSPO in large enterprises . I believe TOGAF (https://www.opengroup.org/) could be a good candidate organization to partner with.  I discussed about TOGAF business architecture when I co-presented with Rob at FINOS  https://osr.finos.org/docs/presentations/strategy

**Comment by winterrocks** [comment]
I agree with @victorjunlu this is a great initiative. Unfortunately I was not able to join the call on Thursday 2024-05-16, but I would like to participate and contribute if/when possible. 

OSPOs are too often seen as a cost and having good concrete examples of how OSPOs can help organizations to save money or create new revenues, etc. is very important, IMHO.

**Reply by llwp** [reply]
Agreed on "too often seen as a cost" - I think there is ongoing work around TCO and related alignment needed. Lots of other matters covered in this expanding work. Exciting times, friends.

**Comment by cm-howard** [comment]
I'm definitely keen to support this and contribute in on this topic; this is very much the area I've been speaking on for the last 2 years or so around demonstrating ROI / Value of OSPOs much like other business functions.

Additionally very much agree that getting the voice/support of C-Suites in Orgs that recognize the value of Open Source above and beyond the 'benefits' would provide a Non-OSPO centric view. 👍

**Comment by lhawthorn** [comment]
What I would say in support of this project has already been said by other commenters, so I will not repeat them. Great idea and thank you so much to those who proposed the idea! (I'd name you to give you full credit, but Chatham House rules. :)

I think one of the most important pieces of feedback to collect is which 3rd party testimonial/imprimaturs/voices would be considered of greatest value in persuading the C-Suite and other critical decision makers. I think that the answer is that this would vary by industry and topic area - some may be more persuaded by academics with domain expertise in a particular research area, some by industry trade organizations, some by other C-Suite/leadership folks in their same industry. In an ideal universe, we could get testimonials or supporting quotes from a wide variety of sources (geographical, industry vertical, social pillar e.g. academia or PubSec if that's a key market for one's business).

I am so excited to help with this project!

**Comment by ar4mirez** [comment]
+10000 to colleagues' comments—This is a much-needed initiative, especially now that open source has been in the news. Having something like this to help the C-Suite see the real value of open source and how to properly adopt it for mutual benefit will strengthen the relationship between open source and business.

As mentioned in the call, I fully support this initiative and would like to be involved.

**Comment by tenatzen** [comment]
Happy to contribute, bringing some of the insight we have from our own efforts in open source development collaboration

**Comment by gvlx** [comment]
ESOP (Open Source Business Portugal) runs an award since 2009 to promote organizations with projects and initiatives in Open Source technologies.

This year we are contacting all the organizations that were involved in the past editions to gather their evolutions and lessons learned.

We would like to know about similar initiatives in the world.

**Comment by anajsana** [comment]
ICYMI, During today's Touchpoint, we are presenting a draft outline to kick off this, based on the group discussions during June's call

**Reply by anajsana** [reply]
Here is the initial outline proposed, with a set of questions/topics brainstormed and grouped by the participants during yesterday's call:


Glossary and Background - Existing frameworks and models | Engineering Driven vs. Business Driven Open Source | Misconceptions and Challenges | Communicating Engineering Benefits to Business Value | Case Studies and Success Stories | Future work and conclusion
-- | -- | -- | -- | -- | --
Guide's purpose and audience | Definitions | Misconceptions managers have about open source | How to communicate open source benefits to business managers | Neutral collaboration: Stories from organizations like LF Energy’s OpenSTEF | Creating a living document that evolves with community input
Open source in modern business | Implementation strategies for business-driven | Addressing security concerns and support availability | Simplifying complex concepts for managerial understanding | Real-life examples to cluster open source suppliers to assess if they are open source first | Recap of the guide’s key points
Carl-Eric Mols' “Principles for Industrial Open Source” | Connecting the two perspectives for a holistic approach | Cyber Resilience Act (CRA) when publishing software | resources for managers to engage with open source effectively | Real-life examples that shares a system that enables organizations  to asses “What do I want to reach by open-sourcing this?” |  
Magnus presentation at FOSSNorth |   | How to properly budget or allocate resources for OSPOs? |   |   |  
Mozilla archetypes and Heather Meeker’s frameworks |   |   |   |   |  

### Notes, curiosities and questions by attendees on the call:
Open Source Program Offices vs InnerSource Program Offices? 
OSPOs as a strategy center for AI (both creation of product and creating internal organizational guidance on generative AI)
Same story for Secure Supply Chain / SBOM
(Reference to OpenChain Project)
Maybe there is a need to build a glossary for this guide to tackle some of the definitions? (e.g OSPO vs ISPO) - LH says “yes, please”
Related to the below: OSPO as a center for strategy on application of public policy as it impacts the business, liaison to/steward in communities where standards to enact policy are set
Challenge: Do you want to be a software producer? (https://www.cyberresilienceact.eu/)
What is the impact? Are there alternatives (join Linux Foundation)?
Context: when setting an OSPO you need to deal with regulations like the CRA. maybe include documentation on what that entails for an OSPO perspective.
https://openchainproject.org/news/2024/04/23/webinar-cyber-resilience-act 
https://openforumeurope.org/news/page/2/ 
How to properly budget or allocate resources for OSPOs?
The OSPOs will take time, so there needs to be multi-year budget
Have a system that enables you to asses “What do I want to reach by open-sourcing this?”-  Are there any open materials that share real life examples of these are done?
Cluster open source suppliers to asses if they are open source first. Are there any open materials that share real life examples of these are done?

## What's next?

The idea is to keep working async on this, either by adding new segments / categories to the outline, changing existing ones, or including new themes to the categories

**Comment by anajsana** [comment]
We will plan to move forward with this proposal in the next TODO Europe Chapter Call. This is scheduled for next September, Thursday 26th from 1pm to 2pm CEST, and you are more than welcome to join and invite your peers interested to contribute to this initiative.

- Agenda is shared in both, [GH discussions](https://github.com/todogroup/ospology/discussions/506) and [google doc](https://docs.google.com/document/d/1r2ZdH2WmA8qZAtR8AxbR108NxgK3-_3ksISSoUWmyDg/edit?usp=sharing)
- [Link to the call](https://www.google.com/url?q=https://zoom.us/j/98966962818?pwd%3D8exKHF8RV864kxtjcWzFVYclWWSKZW.1&sa=D&source=calendar&ust=1727190157212753&usg=AOvVaw0TEji5uo_sNh2swBqj5GF5)
- Check TODO [Calendar](https://todogroup.org/community/meetings/#calendar) to download in iCal format

Proposed areas to cover in this section are:

## Open Source in Business Collaborative Guide - What’s next?
1. Recap on the outline that participants worked on in the [past call](https://github.com/todogroup/ospology/discussions/481#discussioncomment-10157135)
- Glossary and Background 
- Existing frameworks and models
- Engineering Driven vs. Business Driven Open Source 
- Misconceptions and Challenges
- Communicating Engineering Benefits to Business Value
- Case Studies and Success Stories
- Future work and conclusion

2. Define workstreams/cluster of tasks and give them an order/priority. Make sure there are “owners” for each workstream
3. Discuss if further follow-ups should be done in these meetings or need to allocate specific catch up calls/official working group under TODO

**Comment by anajsana** [comment]
Hello all! As discussed during TODO Touchpoint yesterday's call, the SC approved the creation of a working folder under TODO to initiate async collaboration to the open source in business whitepaper / guide. Here is the initial folder (under OSPOlogy) https://github.com/todogroup/ospology/tree/main/whitepapers/business-value

I also created a new #wg-oss-in-business channel in TODO slack to centralize real-time chat discussions. If requested by participants, a groups io working group mailing list can be created too.

A few things that we were not able to do during the call that needs to be defined to move forward with the whitepaper section creation based on the proposed outline:

- Define workstreams/cluster of tasks and give them an order/priority. Make sure there are “owners” for each workstream
- Discuss if further follow-ups should be done in these meetings or need to allocate specific catch up calls/official working group under TODO

**Reply by winterrocks** [reply]
Unfortunately, I was not able to join the call on 2024-09-26, but I am reading comments, etc. now. 

I think that one important thing that we need to keep in mind and that should be clear to managers and business leaders is that that even Open Source components one can normally use without a fee, there are still costs related to the use and those are for example keep the components up-to-date, upstream bug fixes, dealing with the Open Source project will take someone's time, etc. 

Another thing, maybe a good selling point, is the cybersecurity and licensing risks. If one or the other risk is realized, one may be in a situation where a product or service needs to be taken out of the market and I am sure that the business leaders will understand what that can mean. Business continuity in case a risk is realized and how to lower the risk to its minimum?

**Comment by anajsana** [comment]
Good points, @winterrocks ! Perhaps this can be explored further in the next TODO Touchpoint & Open Office Hours Call (scheduled for Thursday, October 31st, from 12:00 to 13:00pm CEST). The call could provide an opportunity for those willing to contribute to this guide to give a 5-minute introductory presentation on the core topic, followed by a space to continue these discussions among participants on the call. The notes from these conversations can later be added as content in the working repository here:https://github.com/todogroup/ospology/tree/main/whitepapers/business-value


Would you (or anyone from this discussion thread) be interested in giving a 5-minute presentation? What do you think about this process?

**Comment by anajsana** [comment]
ICYMI, the agenda for the next touchpoint call where we will keep advancing on this guide has been added to GH discussions. People can check it here: https://github.com/todogroup/ospology/discussions/512

Date: Thursday, October 31
Time: 13:00 pm - 14:00 pm CET (*)
Agenda: [GH Discussion](https://github.com/todogroup/ospology/discussions/512) or [google goc](https://docs.google.com/document/d/1r2ZdH2WmA8qZAtR8AxbR108NxgK3-_3ksISSoUWmyDg/edit?usp=sharing)
Platform: [Zoom](https://zoom.us/j/98966962818?pwd=8exKHF8RV864kxtjcWzFVYclWWSKZW.1)

* Please note that with the switch to winter time in Europe, the meeting time this time is now CET (GMT +01). Please be sure to adjust your schedule accordingly 🙏

**Comment by anajsana** [comment]
Something to discuss maye during this Thursday call. FOSSBackstage CFP is closing on Nov 4th. there is a track dedicated to about how to increase awareness and adoption of Open Source in corporations. https://25.foss-backstage.de/call-for-participation/

Would anyone form this group be interested to send a panel proposal to talk about this topic and idea there? I can prepare a draft and title if there is enough participants and interest! I'm definitely in 🙋‍♀️

**Comment by anajsana** [comment]
People can track progress made during the past Touchpoint Europe Call by reading the meeting notes: https://github.com/todogroup/ospology/discussions/512#discussioncomment-11168620

**Comment by SDKAAA** [comment]
Is there a place where we can suggest "Case Studies and Success Stories" ?

**Reply by anajsana** [reply]
Hello @SDKAAA !  The idea is to have a markdown file for each column of our [outline](https://github.com/todogroup/ospology/tree/main/whitepapers/business-value#initial-outline) in the working repo to populate with content. You can see the community defined a entire section for "Case Studies and Success Stories", if you have any proposal to add there in mind, you can either open a PR initiating a md called `case-studies.md` or start the conversation first here for feedback. I'd also suggest to join our slack channel in [TODO #WG-OSS-IN-BUSINESS](https://join.slack.com/t/thetodogroup/shared_invite/zt-2w71kclgx-JOUB4LTXIuVEKehkJk7V0w)

**Comment by anajsana** [comment]
Hello folks! ICYMI, @harmonyelendu, and I are working on a few documents to include in the oss in business WG to define the roadmap and contributor roles to start in 2025. People can add comments and participate in the PRs here: 
- https://github.com/todogroup/ospology/pull/525
- https://github.com/todogroup/ospology/pull/526

Also, Harmony just started filling out the glossary.md with a few terms, and people can also review and add additional terms here: https://github.com/todogroup/ospology/pull/527

**Comment by anajsana** [comment]
Hello folks!

Here are a few updates on the progress of this guide and the next steps to kick off the activity in 2025 based on the outline and challenges charged in 2024:

# Project Kanban
A project view has been created with all the sections to be covered, helping contributors organize areas where they’d like to contribute: [Project View](https://github.com/orgs/todogroup/projects/10/views/1).

The structure supports milestones that encompass a set of tasks organized as GitHub issues. Based on the contributing file (see below), the idea is to assign a chair(s) and a reviewer(s) for each milestone to coordinate efforts and facilitate progress on each section.

# Roles: Chais, Reviewers and Core Contributors

[Roadmap](https://github.com/todogroup/ospology/blob/main/whitepapers/business-value/Roadmap.md) and [contributing guidelines](https://github.com/todogroup/ospology/blob/main/whitepapers/business-value/CONTRIBUTING.md) have been created and are available in the WG.

> Note: We are currently looking for reviewers and chairs to coordinating efforts and facilitating the progress on each of the sections / chapters of this guide. Please answer to this commend if you would be interested to be assigned as chair / reviewer. 

# Next Sync Call
The next sync call will take place during the upcoming TODO Touchpoint on Thursday, February 13 at 12:00pm GMT. [RSVP here](https://zoom-lfx.platform.linuxfoundation.org/meeting/93266151738?password=e76e3320-7fb0-413a-b01b-33a11733e026).

**Comment by anajsana** [comment]
## Next OSS in Business Guide Sync Call

> Where: TODO Touchpoint of 2025 – February 13th (EMEA & APAC) - RSVP Required

We are excited to invite participants from this working group to the first TODO Touchpoint of 2025, scheduled for Thursday, February 13th, where we have secure space to discuss recent updates, blockers (issues and PRs that needs review), 2025 roadmap and next steps to keep advancing in the OSS in business guide

Agenda:

- SBOM Journey for an Open Source Project: Apache NuttX RTOS
  - Presenter: Alin Jerpelea, OSPO at Sony
  - Duration: 15 minutes presentation followed by 8 minutes for Q&A
- OSS in Business and OSPO Book Working Group Updates - Duration: 15 minutes
- Open Forum and Q&A under Chatham House Rules - Duration: 20 minutes

RSVP Required before the meeting starts: To confirm your participation, please RSVP by clicking the link below: [RSVP Here](https://zoom-lfx.platform.linuxfoundation.org/meeting/93266151738?password=e76e3320-7fb0-413a-b01b-33a11733e026). 

We look forward to your participation 😊

**Comment by anajsana** [comment]
Thanks for joining the Touchpoint yesterday! (recording is [here](https://youtu.be/Aj84quEIEug) and WG discussion start in min 12:22)
As a summary from the discussion and updates shared:

- The [issue](https://github.com/orgs/todogroup/projects/10/views/1?pane=issue&itemId=94058933&issue=todogroup%7Cospology%7C544) from the Milestone "Strategies to implement open source in business" (available in our  [project view](https://github.com/orgs/todogroup/projects/10/views/1)) has some new activity and contributions from participants
- The [contributing guidelines](https://github.com/todogroup/ospology/blob/main/whitepapers/business-value/CONTRIBUTING.md) introduced proposes three different roles to effectively drive group progress: Group participants, chairs and reviewers
- The next step is to identify people in this WG, ideally with experience in github issues, project and community management, willing to step up as CHAIRS of this working group.


## WG Chair Application Form

This form will be open until our next Touchpoint call, Monday 24th, February 

👉 [Take the form](https://forms.gle/NFu2c3brxRBHc2Jz9)
